### PR TITLE
Update _config.yml

### PR DESCRIPTION
**Pull request by michalwosk**
Update nav bar items in docs

 

## Change Description
1. Add "guides" link to nav bar
2. Change main action to be "Try without installing" like in main website
3. Change "community" to slack to improve conversions
4. Change "Docs" to "Home" for clarity

### Background

Refresh on nav bar after a long time it wasn't updated

### Breaking Change?

No

### CODE REVIEW

**Review by eladlachmi — APPROVED** [review]
LGTM

File: docs/_config.yml [hunk 1]
```diff
@@ -17,18 +17,20 @@ remote_theme: pmarsceill/just-the-docs@v0.4.1
 
 # Aux links for the upper right navigation
 aux_links:
-  'Docs':
+  'Home':
     - 'https://docs.lakefs.io/'
```
Comment (rmoff) [inline docs/_config.yml:21]: Do we even need this link? It seems redundant. `Home` is ambiguous and could be assumed to mean lakeFS.io. I'd suggest we just drop this link.

File: docs/_config.yml [hunk 2]
```diff
@@ -40,7 +42,7 @@ footer:
     - 'https://www.treeverse.io/'
 
 footer_links:
-  'Docs':
+  'Home':
     - 'https://docs.lakefs.io/'
```
Comment (rmoff) [inline docs/_config.yml:46]: as above. let's just delete it.

File: docs/_config.yml [hunk 3]
```diff
@@ -17,18 +17,20 @@ remote_theme: pmarsceill/just-the-docs@v0.4.1
 
 # Aux links for the upper right navigation
 aux_links:
-  'Docs':
+  'Home':
     - 'https://docs.lakefs.io/'
   'Blog':
     - 'https://lakefs.io/blog/'
-  'Community':
-    - 'https://lakefs.io/community/'
```
Comment (rmoff) [inline docs/_config.yml:25]: Why are we replacing it? Community is not just Slack alone. @adipolak WDYT?

**Review by rmoff — CHANGES_REQUESTED** [review]

File: docs/_config.yml (hunk 3 above)
Comment (adipolak) [reply docs/_config.yml:25]: agree.

File: docs/_config.yml [hunk 4]
```diff
@@ -17,18 +17,20 @@ remote_theme: pmarsceill/just-the-docs@v0.4.1
...
+    - 'https://lakefs.io/slack/'
   'GitHub':
     - 'https://github.com/treeverse/lakeFS'
-
+    
 buttons:
-  'lakeFS Cloud':
-      - 'https://lakefs.io/cloud-registration/'
+  'Try without installing':
+    - 'https://demo.lakefs.io/'
```
Comment (adipolak) [inline docs/_config.yml:33]: Hi @michalwosk - from what I understand [demo.lakefs.io](http://demo.lakefs.io/) is the old playground environment, the new one is not released to production yet, yet there is a plan to release it today and destroy the old one next week. which other button would you like to have there instead?

File: docs/_config.yml (hunk 1 above)
Comment (michalwosk) [reply docs/_config.yml:21]: I agree. Removing it.

File: docs/_config.yml (hunk 4 above)
Comment (michalwosk) [reply docs/_config.yml:33]: Yes, we will redirect in any case. No worries.

File: docs/_config.yml (hunk 3 above)
Comment (rmoff) [reply docs/_config.yml:25]: Yes, let's keep it as the community page please. I think it's a good page - it doesn't have any marketing 'smells' if you see what I mean :-) 
(well if I was being really fussy I'd remove webinars from `events`, but that's all ;-) )

Comment (michalwosk) [reply docs/_config.yml:25]: Okay, no problem.

Comment (michalwosk) [reply docs/_config.yml]: I changed it to the right link :-)

File: docs/_config.yml [hunk 5]
```diff
@@ -17,18 +17,18 @@ remote_theme: pmarsceill/just-the-docs@v0.4.1
...
 aux_links:
-  'Docs':
-    - 'https://docs.lakefs.io/'
   'Blog':
     - 'https://lakefs.io/blog/'
+  'Guides':
+    - 'https://lakefs.io/guides/'  
   'Community':
-    - 'https://lakefs.io/community/'
+    - 'https://lakefs.io/developers/'
```
Comment (rmoff) [inline docs/_config.yml:25]: Can we leave this unchanged please? 

```suggestion
    - 'https://lakefs.io/community/'
```

**Review by rmoff — CHANGES_REQUESTED** [review]

**Comment by nopcoder** [comment]
@ozkatz is it relevant or can we close this one?

**Pull request closed without merging**